src/pyo3_mock_data/pyo3_mock_data/_pyo3_mock_data.pyi

```diff
@@ -3,5 +3,5 @@ from __future__ import annotations
 from pathlib import Path
 
 def generate_fake_persons(
```

Comment:
Can we call this "mock" persons instead? We should remove all mentions of the word "fake" and be consistent with the parent direct name.

Reply:
Yes, I thought I caught all of these, but apparently I missed some. Will do a global search to make sure I didn't miss more.

Reply:
wow, looks like the one I missed just the function name...the most important one.